OpenImis.ePayment/Logic/PaymentLogic.cs

```diff
@@ -212,6 +213,30 @@ public async Task<DataMessage> SavePayment(PaymentData model)
 #endif*/
 
             }
+            else 
+            {
+                string rejectedReason = "";
+                switch (response.Code) 
+                {
+                    //3 - Wrong Control Number
+                    case 3:
+                        rejectedReason = payment.PrepareRejectedReason(payment.PaymentId, GepgCodeResponses.GepgResponseCodes["Bill does not exist"].ToString());
```

Comment:
We cannot use GepgCodeResponses here as this is part of the generic implementation. The error message must be handled in the escape part, either in PaymentController or ImisPayment in the code between `#if CHF` and `#endif` as only this part is specific for TZ.